clinical-mongo-backup/templates/cronjob.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "clinical-mongo-backup.fullname" . }}
+  labels:
+    {{- include "clinical-mongo-backup.labels" . | nindent 4 }}
+spec:
+  concurrencyPolicy: Forbid
+  schedule:  {{ .Values.schedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: backup
+            image: {{ .Values.image.name }}
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            env:
+            - name: MONGO_HOST
+              value: "{{ .Values.backupConfigs.MONGO_HOST }}"
+            - name: MONGO_PORT
+              value: "{{ .Values.backupConfigs.MONGO_PORT }}"
+            - name: BACKUP_ID
+              value: "{{ .Values.backupConfigs.BACKUP_ID }}"
+            - name: MONGO_DATABASE
+              value: "{{ .Values.backupConfigs.MONGO_DATABASE }}"
+            - name: VAULT_SECRET_PATH
+              value: "{{ .Values.backupConfigs.VAULT_SECRET_PATH }}"
+            - name: VAULT_K8_ROLE
+              value: "{{ .Values.backupConfigs.VAULT_K8_ROLE }}"
+            - name: VAULT_ADDR
+              value: "{{ .Values.backupConfigs.VAULT_ADDR }}"
+            - name: CLINICAL_MONGO_CTL_API
+              value: "3"
+            - name: NFS_USER_ID
+              value: "{{ .Values.nfsUserID }}"
+            - name: RETENTION
+              value: "{{ .Values.retention }}"
+            command: ["/bin/sh"]
+            args: ["/etc/clinical-mongo-backup-script/backup.sh"]
```

Comment:
A unix thing, but `etc` should only ever contain system level configuration files not user added scripts or an executable. 

The proper pace for the backup script would be in `/usr/local` or `/opt`